### PR TITLE
TA-143 functional tests fix

### PR DESCRIPTION
**Pull request by phaniperla**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
See next unanswered question test has been fixed
https://tools.hmcts.net/jira/browse/TA-143

## Type of change

Please delete options that are not relevant.

- [ X] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ruban72 — APPROVED** [review]

**Review by marianadpereira — APPROVED** [review]

File: test/end-to-end/paths/reasonsForDivorce.js [hunk 1]
```diff
@@ -74,7 +78,13 @@ Scenario('Basic Divorce E2E - with added examples', async function(I) {
   } else{
     await I.checkMyAnswersAndValidateSession();
   }
-  I.amDoneAndSubmitted();
+
+  const genericErrorPage = await I.checkElementExist('//h1[contains(text(), \'There has been a problem\')]');
+  if(genericErrorPage) {
+    I.checkGenericErrorPage();
+  }else {
```
Comment (mounikahmcts) [inline test/end-to-end/paths/reasonsForDivorce.js:85]: spacing lint issue

File: test/end-to-end/paths/reasonsForDivorce.js [hunk 2]
```diff
@@ -106,7 +116,14 @@ Scenario('2 years separation E2E', async function(I) {
   } else{
     await I.checkMyAnswers();
   }
-  I.amDoneAndSubmitted();
+
+  const genericErrorPage = await I.checkElementExist('//h1[contains(text(), \'There has been a problem\')]');
+  if(genericErrorPage) {
```
Comment (mounikahmcts) [inline test/end-to-end/paths/reasonsForDivorce.js:121]: same here

File: test/end-to-end/paths/reasonsForDivorce.js [hunk 3]
```diff
@@ -134,5 +151,12 @@ Scenario('5 years separation E2E', async function(I) {
...
```
Comment (mounikahmcts) [inline test/end-to-end/paths/reasonsForDivorce.js:156]: same here

File: test/end-to-end/paths/aboutYourMarriageCertificate.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 const content = require('app/steps/marriage/about-your-marriage-certificate/content.json').resources.en.translation.content;
 
-Feature('Foreign Marriage Certificates - Certificate Language').retry(3);
+Feature('Foreign Marriage Certificates - Certificate Language @functional').retry(3);
```
Comment (mounikahmcts) [inline test/end-to-end/paths/aboutYourMarriageCertificate.js:3]: why is this @functional added in the description?  
In the logs, we can see the description under the filename
Screenshot 2020-10-27 at 18 19 58

File: test/end-to-end/paths/reasonsForDivorce.js (hunk 2 above)
Comment (phaniperla) [reply test/end-to-end/paths/reasonsForDivorce.js:121]: I reviewed  above code and i see it has only one line space which is acceptable in Lint .

File: test/end-to-end/paths/reasonsForDivorce.js (hunk 1 above)
Comment (phaniperla) [reply test/end-to-end/paths/reasonsForDivorce.js:85]: I reviewed  above code and i see it has only one line space which is acceptable in Lint .

File: test/end-to-end/paths/aboutYourMarriageCertificate.js (hunk 4 above)
Comment (phaniperla) [reply test/end-to-end/paths/aboutYourMarriageCertificate.js:3]: @functional is a tag,  At the moment it’s running all tests in regression pack

and also the tag prevents duplicate tests execution previously i noticed “Basic divorce path” is running twice

now that has been excluded from the regression test suite

File: test/end-to-end/paths/reasonsForDivorce.js (hunk 1 above)
Comment (mounikahmcts) [reply test/end-to-end/paths/reasonsForDivorce.js:85]: looks like the project is not setup to check this lint issue. but the standard is https://standardjs.com/rules.html
Screenshot 2020-10-28 at 09 54 10

**Review by mounikahmcts — APPROVED** [review]

**Review by mhisltd — APPROVED** [review]
LGTM - Approving the change

**Pull request merged**